app/controllers/chatrooms_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ChatroomsController < ApplicationController
+  
+  def new
+    @chatroom = Chatroom.new
+  end
+
+  def create
+    @chatroom = Chatroom.new chatroom_params
+    if @chatroom.save
+      flash[:info] = t ".controller.chatrooms.create_chatroom_succ"
+      redirect_to new_chatroom_path
+    else
+      render :new
+    end
+  end
+
+  def destroy
```

Comment:
I don't see the use of this action! :))

Reply:
my partner do that in his branch. we'll complete it in next pull request!